grapher/scan.py

```diff
@@ -106,6 +125,23 @@ def pkgToUnit(pkg: Dict) -> Unit:
             ReqFiles = [normalize(os.path.join(pkgdir, "requirements.txt"))],
         )
     )
+    if len(tests) == 0:
+        return [unit]
+    return [unit, Unit(
+        Name = pkg['project_name'],
+        Type = TEST_UNIT_KEY.Type,
+        Repo = "",
+        CommitID = "",
+        Files = sorted(tests),
+        Dir = TEST_DIR,
+        Dependencies = [UnitKey(
```

Comment:
can you use `unit.key()` here or something like
```
Name = unit.Name,
Type = unit.Type,
Repo = unit.Repo,
CommitID = unit.CommitID,
Version = unit.Version,
```

That way, if anything changes to unit, you won't also have to update this code.